include/regs.h

```diff
@@ -67,29 +67,6 @@
 #define R_FB_FILTER_ENV_COLOR(c)                 SREG(85 + (c))
 #define R_ENABLE_FB_FILTER                       SREG(88)
 #define R_PAUSE_BG_PRERENDER_STATE               SREG(94) // `PauseBgPreRenderState`
-#define R_CAM_XZ_OFFSET_UPDATE_RATE              OREG(2)
-#define R_CAM_Y_OFFSET_UPDATE_RATE               OREG(3)
-#define R_CAM_FOV_UPDATE_RATE                    OREG(4)
-#define R_CAM_MAX_PITCH                          OREG(5)
-#define R_CAM_R_UPDATE_RATE_INV                  OREG(6)
```

Comment:
Not a fan of losing all the oreg names from this file, but I dont have any better ideas. I do agree that keeping the camera-specific data defined in here would be even weirder so this seems like the lesser of two evils

Reply:
You can always leave the symbol names in for documentation purposes